op-mode-definitions/show-virtual-server.xml.in

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="virtual-server">
+        <properties>
+          <help>Show Virtual Server information</help>
```

Comment:
I don't think we should capitalize "Virtual Server" here.

Reply:
Done